cves/kernel/CVE-2013-3223.yml

```diff
@@ -221,8 +259,9 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
+  name: net
   note:
```

Comment:
It would be nice if there was a note to describe this subsystem a bit more

Reply:
Will do